receiver/k8sobjectsreceiver/testdata/e2e/collector/configmap.yaml

```diff
@@ -29,10 +29,11 @@ data:
         log_statements:
           - context: log
             statements:
-              - replace_match(body["object"]["metadata"]["uid"], "*", "12345")
-              - replace_match(body["object"]["metadata"]["resourceVersion"], "*", "12345")
-              - replace_match(body["object"]["metadata"]["creationTimestamp"], "*", "2000-01-01T00:00:00Z")
-              - replace_match(body["object"]["metadata"]["managedFields"][0]["time"], "*", "2000-01-01T00:00:00Z")
+              - replace_match(body["metadata"]["uid"], "*", "12345")
```

Comment:
This looks like a breaking change to me. Is this something that we want to introduce?